Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/08-RecordMeeting/RecordMeetingFeature-02-code-0003-previous.swift

```diff
@@ -64,10 +64,9 @@ struct SyncUpDetailView: View {
       }
 
       Section {
-        Button("Delete") {
+        Button("Delete", role: .destructive) {
```

Comment:
📓 These changes are here to match the changes made in previous part of the tutorial